airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json

```diff
@@ -88,6 +88,15 @@
         }
       }
     },
+    "bid_strategy" : {
+      "type": ["null", "string"]
+    },
+    "bid_amount": {
+      "type": ["null", "number"]
+    },
+    "bid_constraints": {
+      "type": ["null", "object"]
```

Comment:
Can you define the schema of the `bid_constraints` object please?

Reply:
Sure!

Reply:
Done :tada: